docker/web/Dockerfile

```diff
@@ -13,13 +13,14 @@
 #
 # WARNING: the shendoah image is a nightly, untested, experimental build. If
 # you want to use an official openjdk image instead use the web-and-data image.
-FROM maven:3-openjdk-11 as build
+FROM maven:3-openjdk-17 as build
 COPY $PWD /cbioportal
 WORKDIR /cbioportal
 ARG MAVEN_OPTS=-DskipTests
 RUN mvn ${MAVEN_OPTS} clean install
 
-FROM shipilev/openjdk:11
+# We shouldn't use openjdk images. See here for explanation: https://hub.docker.com/_/openjdk
+FROM eclipse-temurin:17
```

Comment:
Does this offer Shenandoah as well? We will need to double check the parameters and confirm the behavior is still the same for production since we are using that garbage collector there CC @dippindots. We could switch to another low pause GC if preferred but we would need to test its performance

Reply:
Good question - I checked and it looks like we're all set. Here are some relevant log statements:

```
cbioportal-container           | Running: /bin/sh -c java -XX:+UnlockExperimentalVMOptions -XX:+UseShenandoahGC -XX:ShenandoahUncommitDelay=1000 -XX:ShenandoahGuaranteedGCInterval=10000 -Dauthenticate=noauthsessionservice -Dsession.service.url=http://cbioportal-session:5000/api/sessions/my_portal/ -jar webapp-runner.jar -AmaxHttpHeaderSize=16384 -AconnectionTimeout=20000 --enable-compression /cbioportal-webapp
cbioportal-container           | Connector attributes
cbioportal-container           | property: maxHttpHeaderSize - 16384(OK)
cbioportal-container           | property: connectionTimeout - 20000(OK)
...
cbioportal-container           | May 31, 2022 2:07:51 PM org.apache.coyote.AbstractProtocol start
cbioportal-container           | INFO: Starting ProtocolHandler ["http-nio-8080"]
```